release_docs/NEWSLETTER.txt

```diff
@@ -3,7 +3,12 @@ HDF5 version 1.14.5-2 currently under development
 Features included for the next major release:
 ----------------------------------------------------------------------------
 
-*
+* The release process now provides signed Windows and Mac installation binaries 
+  in addition to the Debian and rpm installation binaries. 
+* Added new routines for interacting with error stacks: H5Epause_stack, 
```

Comment:
"The mac binaries are built as universal binaries on an arm-based mac."
Should be added.